internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -53,10 +50,12 @@ spec:
       description: JSON build information for the requested build
     - name: buildState
       description: IIB Service build state
+    - name: misc
```

Comment:
why `misc`? what does that mean?

Reply:
I had to add a result to transfer the opt in information to the internalrequest. In the `create-internal-request` it is called also called`misc` therefore I kept the same name.